portal/templates/initial_queries.html

```diff
@@ -99,38 +88,76 @@ <h4 class="profile-item-title"> {{ _("Thank you.") }}</h4>
 
 var fc, mainSections = {}, _isChecked = false, rolesGroupClickEvent, patientQClickEvent, userOrgChangeEvent;
 
-var CONFIG_REQUIRED_CORE_DATA = {%if config.REQUIRED_CORE_DATA %}{{config.REQUIRED_CORE_DATA|safe}}{% else%}false{%endif%};
-
-function inConfig(configMatch) {
-  if (! hasValue(configMatch)) return false;
-  else {
-    if (CONFIG_REQUIRED_CORE_DATA) {
-      var found = false;
-      var ma = configMatch.split(",");
-      ma.forEach(function(item) {
-        CONFIG_REQUIRED_CORE_DATA.forEach(function(v) {
-            if (!found && v == item) found = true;
-        });
-      });
-      return found;
-    } else return true;
+/**** object for obtaining required core data ****/
+var configObj = (function() {
+  var CONFIG_DEFAULT_CORE_DATA = {%if config.REQUIRED_CORE_DATA %}{{config.REQUIRED_CORE_DATA|safe}}{% else%}false{%endif%};
```

Comment:
Paul, not sure if my previous comment made it to you.  Typed it quick before the meeting.  I was saying that I actually need it for the progressbar - which shows the state of each section (whether it is completed)